### ISSUE TITLE
max and min don't support key arguments

### ISSUE THREAD

**Issue by voraseth** [skulpt/skulpt#393]
The following code cannot run because keyword arguments are not supported.

```
a = [3,1,4,7,2]
print(max(range(len(a)), key=lambda i: a[i]))
```

**Comment by waywaaard** [comment]
Actually, the max/min just do not support the optional ```key``` argument.

**Comment by eah13** [comment]
@voraseth fair to change this issue to "max and min don't support key arguments" for clarity?

**Comment by s-cork** [comment]
closed via #1065
